docker/Dockerfile

```diff
@@ -50,11 +42,10 @@ ENV REACT_APP_ENV=$REACT_APP_ENV_ARG
 ENV REACT_APP_CODECOV_VERSION=$REACT_APP_CODECOV_VERSION
 ENV GENERATE_SOURCEMAP=false
 RUN mkdir /home/workspace
-COPY --from=npm /home/workspace/node_modules /home/workspace/node_modules
 WORKDIR /home/workspace
 RUN apk -U add git
-RUN apk add --no-cache yarn
 COPY . /home/workspace
+RUN corepack enable
```

Comment:
Super cool!

Does corepack have access to the package.json even if we're no longer copying it over with `COPY package.json /home/workspace` ?

Reply:
Oh nvm, it's getting copied in above (?). Wonder why it was duplicated..

Reply:
not too sure tbh 🤔